pymc3/data.py

```diff
@@ -124,6 +124,13 @@ class Minibatch(tt.TensorVariable):
     minibatch : minibatch tensor
         Used for training
 
+    Notes
+    -----
+    It is a common case to use a `Minibatch` within variational inference.
```

Comment:
nip:
"Below is a common use case of `Minibatch` within the variational inference. Importantly, we need to make PyMC3 "aware" of minibatch being used in inference. Otherwise, we will get wrong :math:`logp` for the model. To do so, we need to pass `total_size` parameter to the observed node, which correctly scale the density of the model logp that effected by Minibatch. See more in examples below."